src/main/ts/components/Editor.ts

```diff
@@ -62,6 +62,7 @@ const initialise = (ctx: IEditor) => () => {
 
   if (isTextarea(ctx.element)) {
     ctx.element.style.visibility = '';
+    ctx.element.style.display = '';
```

Comment:
We should add a comment here saying what this is for.

Reply:
Looking at the code it appears the Vue is responsible for it being `display: hidden`? It seems that you're removing the attribute before TinyMCE could get a chance to mess with it.

Reply:
Possible cause:
https://www.ryansouthgate.com/2020/01/30/vue-js-v-hide-element-whilst-keeping-occupied-space/#vshow